tests/openapi/openapi_integration/test_query_full.py

```diff
@@ -0,0 +1,551 @@
+            "strategy": "best_score",
+            "using": "dense-image",
+        },
+    )
+    assert response.ok
+    recommend_result = response.json()["result"]
+
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/query",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "query": {
+                "recommend": {
+                    "positive": [1, 2, 3, 4],  # ids
+                    "negative": [3],  # ids
+                    "strategy": "best_score",
+                },
+            },
+            "using": "dense-image",
+        },
+    )
+    assert response.ok
+    query_result = response.json()["result"]
+
+    assert recommend_result == query_result
+
+
+def test_discover():
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/discover",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "target": 2,  # ids
+            "context": [{"positive": 3, "negative": 4}],  # ids
+            "limit": 10,
+            "using": "dense-image",
+        },
+    )
+    assert response.ok
+    discover_result = response.json()["result"]
+
+    query_result = root_and_rescored_query(
+        {
+            "discover": {
+                "target": 2,
+                "context": [{"positive": 3, "negative": 4}],
+            }
+        },
+        "dense-image"
+    )
+
+    assert discover_result == query_result
+
+
+def test_context():
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/discover",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "context": [{"positive": 2, "negative": 4}],
+            "limit": 100,
+            "using": "dense-image",
+        },
+    )
+    assert response.ok
+    context_result = response.json()["result"]
+
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/query",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "query": {
+                "context": [{"positive": 2, "negative": 4}],
+            },
+            "limit": 100,
+            "using": "dense-image",
+        },
+    )
+    assert response.ok
+    query_result = response.json()["result"]
+
+    assert set([p["id"] for p in context_result]) == set([p["id"] for p in query_result])
+
+
+def test_order_by():
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/scroll",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "order_by": "count",
+        },
+    )
+    assert response.ok
+    scroll_result = response.json()["result"]["points"]
+
+    query_result = root_and_rescored_query({"order_by": "count"}, "dense-image", with_payload=True)
```

Comment:
`"dense-image"`?

Reply:
I meant specifying "using" for order by

Reply:
Good catch, forgot that one :face_with_head_bandage: 
Will fix with the next batch of tests :)